ui/src/app/richskill/list/skills-list.component.spec.ts

```diff
@@ -383,29 +388,32 @@ describe("SkillsListComponent", () => {
     expect(action3.callback?.(action3, skill3)).toBeFalsy()  // Always false
     expect(action3.visible?.(skill3)).toBeFalsy()  // != Archived
 
+    const action4 = tableActions[4]
+    expect(action4.label).toEqual("Export Selected")
+
     component.selectedSkills = [
       createMockSkillSummary("id1", PublishStatus.Draft),
     ]
     component.showAddToCollection = true
     tableActions = component.tableActions()
-    let skill4 = createMockSkillSummary("id4", PublishStatus.Archived)
-    let action4 = tableActions[4]
-    expect(action4.label).toEqual("Add to Collection")
-    expect(action4 && action4.callback).toBeTruthy()
-    expect(action4.callback?.(action4, skill4)).toBeFalsy()  // Always false
-    expect(action4.visible?.(skill4)).toBeTruthy()  // There are selected skills
+    let skill5 = createMockSkillSummary("id4", PublishStatus.Archived)
+    let action5 = tableActions[5]
```

Comment:
Must we use just an index value here? Do these actions have identifiers?

Reply:
The code was using index, is necessary to change all the code?
e.g `actionExportSelected` instead of `action4`, `actionAddCollection` instead of `action5`.

Reply:
we should focus on our changes only this time, we should make the other changes in a refactor story later